python/tennicam_client/parser.py

```diff
@@ -0,0 +1,34 @@
+import typing
+import pathlib
+
+Position = typing.TypeVar("Position", float, float, float)
```

Comment:
I'm a bit confused about how `TypeVar` exactly works but it seems that the additional type arguments define a union rather than a list of elements.  From the [docs](https://docs.python.org/3/library/typing.html#typing.TypeVar):

    A = TypeVar('A', str, bytes)  # Must be str or bytes

To represent a tuple of three values, you could use `typing.Tuple[float, float, float]`.  This has the drawback that it really needs to be a tuple in this case (i.e. not a list, np.array, etc.).  Specifying a fixed-size mutable sequence is unfortunately not possible as far as I know.